include/nfc/ndef/uri_msg.h

```diff
@@ -47,7 +47,7 @@ extern "C" {
  */
 int nfc_ndef_uri_msg_encode(enum nfc_ndef_uri_rec_id uri_id_code,
 			    uint8_t const *const uri_data,
-			    uint8_t uri_data_len,
+			    uint16_t uri_data_len,
```

Comment:
Wouldn't be better to use `uint32_t` ? Seems like record payload length is encoded on a 4 bytes

Reply:
Good point. Sure, I can do that

Reply:
Well, I reconsidered it. I think that having a 4GB buffer seems impossible on our device so we might as well stick to uint16_t data type.

Reply:
Theoretically, that would make encoding record bigger than 64kb possible but I am not sure if there will be use cases for such big record.